runelite-client/src/main/java/net/runelite/client/plugins/inventorysetups/InventorySetupPlugin.java

```diff
@@ -288,104 +185,58 @@ private void updateConfig()
 		final String json = gson.toJson(inventorySetups);
 		configManager.setConfiguration(CONFIG_GROUP, CONFIG_KEY, json);
 	}
-
 	private void loadConfig()
 	{
 		// serialize the internal data structure from the json in the configuration
 		final String json = configManager.getConfiguration(CONFIG_GROUP, CONFIG_KEY);
-		if (json == null || json.isEmpty())
+		if (Strings.isNullOrEmpty(json))
 		{
-			inventorySetups.clear();
+			inventorySetups = new ArrayList<>();
 		}
 		else
 		{
-			// TODO add last resort?, serialize exception just make empty map
-			final Gson gson = new Gson();
-			Type type = new TypeToken<HashMap<String, InventorySetup>>()
+			try
 			{
+				final Gson gson = new Gson();
+				Type type = new TypeToken<ArrayList<InventorySetup>>() {
 
-			}.getType();
-			inventorySetups.clear();
-			inventorySetups.putAll(gson.fromJson(json, type));
-		}
+				}.getType();
+				inventorySetups = gson.fromJson(json, type);
+			}
+			catch (Exception e)
```

Comment:
catching all exceptions is bad, I don't know what can be thrown in there but it should be specific-ied down into what is expected to be thrown in there